tests/components/ClaimStatus.test.tsx

```diff
@@ -0,0 +1,41 @@
+import renderer from 'react-test-renderer'
+import { ClaimStatus } from '../../components/ClaimStatus'
+import getScenarioContent, { ScenarioType } from '../../utils/getScenarioContent'
+import apiGatewayStub from '../../utils/apiGatewayStub'
+import { ClaimStatusContent } from '../../types/common'
+
+function getClaimStatusJSON(statusContent: ClaimStatusContent): string {
+  return renderer
+    .create(
+      <ClaimStatus
+        loading={false}
+        statusDescription={statusContent.statusDescription}
+        nextSteps={statusContent.nextSteps}
+      />,
+    )
+    .toJSON()
+}
+
+function testClaimStatus(scenarioType: ScenarioType): void {
+  const scenarioContent = getScenarioContent(apiGatewayStub(scenarioType))
+  const tree = getClaimStatusJSON(scenarioContent.statusContent)
+  expect(tree).toMatchSnapshot()
+}
+
+describe('ClaimStatus', () => {
+  it('renders for Scenario1', () => {
+    testClaimStatus(ScenarioType.Scenario1)
```

Comment:
I guess these can be `expect(testClaimStatus(***)).toMatchSnapshot()`? and return the tree in testClaimStatus

![image](https://user-images.githubusercontent.com/82277/125348228-f3e57e80-e329-11eb-9942-f1fdb43a557b.png)